src/content/app/tools/blast/components/species-selector/SpeciesSelector.scss

```diff
@@ -0,0 +1,123 @@
+@import 'src/styles/common';
+
+.header {
+  display: flex;
+  width: 860px;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 28px;
+  padding-bottom: 12px;
+}
+
+.headerGroup {
+  display: flex;
+  align-items: center;
+}
+
+.headerTitle {
+  font-size: 15px;
+  font-weight: $bold;
+}
+
+.maxSpecies {
+  font-weight: $light;
+  margin-right: 30px;
+}
+
+.clearAll {
+  margin-right: 30px;
+  color: $blue;
+  cursor: pointer;
+  user-select: none;
+}
+
+.speciesCounter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 28px;
+  color: $white;
+  background-color: $dark-grey;
+  font-weight: $bold;
+  border-radius: 100%;
+  font-size: 15px;
+  margin: 0 0.6rem 0 0.6rem;
+}
+
+.speciesSelectorPlaceholder {
+  width: 860px;
+  height: 684px;
+  background-color: $light-grey;
+  border: 1px solid $grey;
+  padding: 10px;
+
+  .checkboxUnchecked {
+    margin-top: 0px;
+    margin-left: 10px;
+    width: 13px;
+    height: 13px;
+  }
+
+  .checkboxChecked {
+    margin-top: 0px;
+    margin-left: 10px;
+    width: 13px;
+    height: 13px;
+  }
```

Comment:
These two checkbox classes should, hopefully, be irrelevant (or at least the checked/unchecked and the width and height part of them should) after Imran's PR gets merged.

Reply:
sure will update my branch once Imran PR is merged.